tests/containers/bci_repo.pm

```diff
@@ -0,0 +1,63 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: bci-repo test
+#   SUSE Linux Enterprise Base Container Images (SLE BCI)
+#   provides truly open, flexible and secure container images and application
+#   development tools for immediate use by developers and integrators without
+#   the lock-in imposed by alternative offerings.
+#
+#   This module is used to test BCI repository.
```

Comment:
Could you add some more info, like:
# We use podman as container engine of reference.
# Podman is assumed to be installed on the host (we can discuss this btw).
# ...

Reply:
oh, that is a big comment that I cannot miss :D